integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -70,16 +75,32 @@ def __init__(self, user, enterprise_configuration):
         """
         super(ContentMetadataExporter, self).__init__(user, enterprise_configuration)
         self.enterprise_api = EnterpriseApiClient(self.user)
+        self.enterprise_catalog_api = EnterpriseCatalogApiClient(self.user)
 
     def export(self, **kwargs):
         """
         Return the exported and transformed content metadata as a dictionary.
         """
         content_metadata_export = {}
-        content_metadata_items = self.enterprise_api.get_content_metadata(
-            self.enterprise_customer,
-            enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
-        )
+        request = crum.get_current_request()
+        if request and waffle.flag_is_active(request, USE_ENTERPRISE_CATALOG):
+            LOGGER.info(
+                'ENT-2572 [1] - Getting catalog metadata for enterprise [%s] from Enterprise Catalog Service.',
+                self.enterprise_customer.name
+            )
+            content_metadata_items = self.enterprise_catalog_api.get_content_metadata(
+                self.enterprise_customer,
+                enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
+            )
+        else:
+            LOGGER.info(
+                'ENT-2572 [2] - Getting catalog metadata for enterprise [%s] from LMS Enterprise.',
+                self.enterprise_customer.name
+            )
+            content_metadata_items = self.enterprise_api.get_content_metadata(
+                self.enterprise_customer,
+                enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
+            )
```

Comment:
Total nit, but you could DRY this up a little bit:
```python
if request and waffle.flag_is_active(request, USE_ENTERPRISE_CATALOG):
    logger_index, metadata_source = (1, "Enterprise Catalog Service")
    metadata_function = self.enterprise_catalog_api.get_content_metadata
else:
    logger_index, metadata_source = (2, "LMS Enterprise")
    metadata_function = self.enterprise_api.get_content_metadata
            
LOGGER.info(
    'ENT-2572 [%s] - Getting catalog metadata for enterprise [%s] from %s.',
    logger_index,
    self.enterprise_customer.name,
    metadata_source
)
content_metadata_items = metadata_function(
    self.enterprise_customer,
    enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
)
```

...having typed that up, it doesn't really save many lines of code, so feel free to completely disregard :)

Reply:
This should be cleaned up once we fully transition to using enterprise-catalog endpoints and remove the use of the waffle flag so I think I'm gonna leave as is for now

Reply:
👍 Make sense.